### PR TITLE
Add ElePHPants

### PR DESCRIPTION
**Pull request by IgorDuarteDev**

### CODE REVIEW

File: src/Elewant/Herding/DomainModel/Breed/Breed.php [hunk 1]
```diff
@@ -26,6 +26,7 @@ final class Breed
     public const BLUE_ZEND_REGULAR = 'BLUE_ZEND_REGULAR';
     public const BLUE_ZEND_ZOE_REGULAR = 'BLUE_ZEND_ZOE_REGULAR';
     public const BLUE_ZRAY_REGULAR = 'BLUE_ZRAY_REGULAR';
+    public const BLUE_PHPCE_REGULAR = 'BLUE_PHPCE_REGULAR';
```
Comment (cookieguru) [inline src/Elewant/Herding/DomainModel/Breed/Breed.php:29]: See #213

**Review by cookieguru — CHANGES_REQUESTED** [review]

**Comment by cookieguru** [comment]
The PHPCon Poland elePHPant is also not officially-licensed

**Comment by IgorDuarteDev** [comment]
are add only the officers?
I think a lot of people have and are interested in these others

**Comment by ramondelafuente** [comment]
Yes. Because we feel it's not fair to encourage unofficial elephpants.

**Comment by IgorDuarteDev** [comment]
Ok, understand.
I closed PR.
Is linux for php ElePHPant is official?

**Comment by cookieguru** [comment]
I would say so, they thanked Vincent in [the announcement tweet](https://twitter.com/linuxforphp/status/1156635964363694080)

**Pull request closed without merging**